wrapping/test/CMakeLists.txt

```diff
@@ -58,6 +58,21 @@ itk_python_add_test(NAME PythonCurvilinearResampleTest
     --first-sample-distance 26.4
   )
 
+itk_python_expression_add_test(NAME PythonInstantiateGaussianInterpolateImageFunctionSSSCI
+  EXPRESSION "iFunc = itk.GaussianInterpolateImageFunction[itk.SliceSeriesSpecialCoordinatesImage[itk.Image[itk.UC, 2], itk.Euler3DTransform[itk.D]], itk.D].New()"
+  )
+
+itk_python_add_test(NAME PythonSliceSeriesResampleTest
+  TEST_DRIVER_ARGS
+  --compareIntensityTolerance 1
+  --compare
+    DATA{${test_baseline_dir}/itkSliceSeriesSpecialCoordinatesImageTestPython.mha}
+    ${ITK_TEST_OUTPUT_DIR}/itkSliceSeriesSpecialCoordinatesImageTestPython.mha
+  COMMAND PythonSliceSeriesResample.py
+    -i DATA{${test_input_dir}/bmode_p59.hdf5}
```

Comment:
I don't have HDFView installed, so I can't check whether this file contains per-slice transforms. Do you have it Tom?

Reply:
I do not.

Reply:
I installed it now from [official site](https://www.hdfgroup.org/downloads/hdfview/), and I can see that it contains extra information.
<img width="1051" alt="bmode_p95_hdf5" src="https://github.com/KitwareMedical/ITKUltrasound/assets/1792121/b6092fb2-cd27-447b-9d84-279ff6fd61d4">

Reply:
I'm unclear, are there transforms within the file? Based on component names it appears to be a rectangular B-Mode image with some metadata for collection angles.

Reply:
I'm not sure I fully understand, does that mean that this is a valid test image? Or not valid?

Reply:
The stuff read is shoved into MetaDataDictionary:
https://github.com/KitwareMedical/ITKUltrasound/blob/v0.6.2/src/itkHDF5UltrasoundImageIO.cxx#L394-L414

I converted this PR into draft until I figure out what to do with it.

Reply:
Only rotation seems to be respected. This is probably easiest to debug by modifying https://github.com/KitwareMedical/ITKUltrasound/blob/master/test/itkSliceSeriesSpecialCoordinatesImageTest.cxx to use `bmode_p59.hdf5` and set the metadata similarly to https://github.com/dzenanz/ITKUltrasound/commit/6ed1737902ed9a53bb870f767de105a01678e1aa

Tom, if you have time to do it, that would be good. I am going on vacation the next two weeks. And I  will be mostly working on a different project today and tomorrow (with pending progress update tomorrow afternoon).

Reply:
@tao558 might pitch in. Coordinate with him while I am away.

Reply:
I found a solution. Problem was missing wrapping for `UltrasoundImageFileReader`. That should be used by the test. I will push the update soon.